build.rs

```diff
@@ -0,0 +1,73 @@
+
+extern crate build_const;
+
+pub fn make_table_crc32(poly: u32) -> [u32; 256] {
```

Comment:
moving `make_table` out of crc32/crc64 is a breaking change since they're public
i'll see weather we could reference them in build.rs after merging.